src/devices/Common/System/Device/Spi/SpiBusInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace System.Device.Spi
+{
+    /// <summary>
+    /// Base class for SPI bus information.
+    /// </summary>
+    public static class SpiBusInfo
+    {
+        private static int s_bufferSize = GetBufferSize();
+
+        /// <summary>
+        /// Buffer size assigned to the SPI bus or -1 if the buffer size is not available.
+        /// </summary>
+        public static int BufferSize => s_bufferSize;
+
+        private static int GetBufferSize()
+        {
+            const string parameterPath = "/sys/module/spidev/parameters/bufsiz";
```

Comment:
can you also add a test similar to https://github.com/dotnet/iot/blob/main/src/System.Device.Gpio.Tests/Mcp3008Tests.cs in the same folder? In the test just do something similar to:

```csharp
if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
{
    Assert.True(SpiBusInfo.GetBufferSize() > 0, "SpiBusInfo.GetBufferSize() returned 0 or less");
}
else
{
    Assert.Equal(-1, SpiBusInfo.GetBufferSize());
}
```

All our Linux tests run on Raspbian so that should always work. This is to make sure that i.e. file doesn't contain any whitespace such as newline character and int.TryParse isn't returning false because of that